app/views/resources/_collection.html.erb

```diff
@@ -8,19 +8,17 @@
 
 <%= content_for :navbar_items do %>
   <% if policy(collection).edit? %>
```

Comment:
We could get rid of this `if` block and add:

``` ruby
def render?
  policy.edit?
end
```

to our component. This is a nice way of controlling when to display your components without having to put any logic in your views.

Reply:
The only issue with this is that then we would have an empty `<li>` hanging out in our HTML since the `<li>` would still render but the button would not.

We could include the `<li>` in the component, but I don't know if I like that, because it arbitrarily assumes that this button will only ever be in a list.

The other thing is that the `if` block also controls the rendering of the settings below, so we would still need it anyway unless we refactored things further.

Reply:
I think solve the <li> issue by using slots (https://viewcomponent.org/guide/slots.html) which would enable us to put the button in different html surroundings. Also, I think the settings button should be refactored into its own component as well, and then could be treated the same as the edit resource. However, this is getting further afield from the ticket, so let's leave it as is for now.